src/Text/Layout/Table/LineStyle.hs

```diff
@@ -80,88 +94,197 @@ unicodeVertical HeavyDash4Line = "┋"
-unicodeTopJoins' :: UnicodeJoin -> UnicodeJoin -> String
-unicodeTopJoins' NoJoin _      = ""
-unicodeTopJoins' _      NoJoin = ""
-unicodeTopJoins' Light  Light  = "┬"
-unicodeTopJoins' Light  Heavy  = "┰"
-unicodeTopJoins' Light  Double = "╥"
-unicodeTopJoins' Heavy  Light  = "┯"
-unicodeTopJoins' Heavy  Heavy  = "┳"
-unicodeTopJoins' Heavy  Double = "╥"  -- No dedicated glyph, use Light Double
-unicodeTopJoins' Double Light  = "╤"
-unicodeTopJoins' Double Heavy  = "╤"  -- No dedicated glyph, use Double Light
-unicodeTopJoins' Double Double = "╦"
-
-unicodeBottomJoins' :: UnicodeJoin -> UnicodeJoin -> String
-unicodeBottomJoins' NoJoin _      = ""
-unicodeBottomJoins' _      NoJoin = ""
-unicodeBottomJoins' Light  Light  = "┴"
-unicodeBottomJoins' Light  Heavy  = "┸"
-unicodeBottomJoins' Light  Double = "╨"
-unicodeBottomJoins' Heavy  Light  = "┷"
-unicodeBottomJoins' Heavy  Heavy  = "┻"
-unicodeBottomJoins' Heavy  Double = "╨"  -- No dedicated glyph, use Light Double
-unicodeBottomJoins' Double Light  = "╧"
-unicodeBottomJoins' Double Heavy  = "╧"  -- No dedicated glyph, use Double Light
-unicodeBottomJoins' Double Double = "╩"
-
-unicodeLeftJoins' :: UnicodeJoin -> UnicodeJoin -> String
-unicodeLeftJoins' NoJoin _      = ""
-unicodeLeftJoins' _      NoJoin = ""
-unicodeLeftJoins' Light  Light  = "├"
-unicodeLeftJoins' Light  Heavy  = "┝"
-unicodeLeftJoins' Light  Double = "╞"
-unicodeLeftJoins' Heavy  Light  = "┠"
-unicodeLeftJoins' Heavy  Heavy  = "┣"
-unicodeLeftJoins' Heavy  Double = "╞"  -- No dedicated glyph, use Light Double
-unicodeLeftJoins' Double Light  = "╟"
-unicodeLeftJoins' Double Heavy  = "╟"  -- No dedicated glyph, use Double Light
-unicodeLeftJoins' Double Double = "╠"
-
-unicodeRightJoins' :: UnicodeJoin -> UnicodeJoin -> String
-unicodeRightJoins' NoJoin _      = ""
-unicodeRightJoins' _      NoJoin = ""
-unicodeRightJoins' Light  Light  = "┤"
-unicodeRightJoins' Light  Heavy  = "┥"
-unicodeRightJoins' Light  Double = "╡"
-unicodeRightJoins' Heavy  Light  = "┨"
-unicodeRightJoins' Heavy  Heavy  = "┫"
-unicodeRightJoins' Heavy  Double = "╡"  -- No dedicated glyph, use Light Double
-unicodeRightJoins' Double Light  = "╢"
-unicodeRightJoins' Double Heavy  = "╢"  -- No dedicated glyph, use Double Light
-unicodeRightJoins' Double Double = "╣"
-
-unicodeInteriorJoins' :: UnicodeJoin -> UnicodeJoin -> String
-unicodeInteriorJoins' NoJoin _      = ""
-unicodeInteriorJoins' _      NoJoin = ""
-unicodeInteriorJoins' Light  Light  = "┼"
-unicodeInteriorJoins' Light  Heavy  = "╂"
-unicodeInteriorJoins' Light  Double = "╫"
-unicodeInteriorJoins' Heavy  Light  = "┿"
-unicodeInteriorJoins' Heavy  Heavy  = "╋"
-unicodeInteriorJoins' Heavy  Double = "╫"  -- No dedicated glyph, use Light Double
-unicodeInteriorJoins' Double Light  = "╪"
-unicodeInteriorJoins' Double Heavy  = "╪"  -- No dedicated glyph, use Double Light
-unicodeInteriorJoins' Double Double = "╬"
+-- | Find the Unicode box-drawing character which joins lines of given weights.
```

Comment:
Ok, I like this much better, including the disclaimer.  Perhaps you could merge those patches somehow or at least for the file `LineStyle`.  I do not see any point in keeping the other one.

Reply:
Will do.